src/Shared/HttpSys/RequestProcessing/NativeRequestContext.cs

```diff
@@ -781,42 +803,42 @@ private uint GetChunksHelper(ref int dataChunkIndex, ref uint dataChunkOffset, b
     {
         if (PermanentlyPinned)
         {
-            return GetClientCertificate((IntPtr)_nativeRequest, (HttpApiTypes.HTTP_REQUEST_V2*)_nativeRequest);
+            return GetClientCertificate((IntPtr)_nativeRequest, (HTTP_REQUEST_V2*)_nativeRequest);
         }
         else
         {
             fixed (byte* pMemoryBlob = _backingBuffer.Memory.Span)
             {
-                var request = (HttpApiTypes.HTTP_REQUEST_V2*)(pMemoryBlob + _bufferAlignment);
+                var request = (HTTP_REQUEST_V2*)(pMemoryBlob + _bufferAlignment);
                 return GetClientCertificate(_originalBufferAddress, request);
             }
         }
     }
 
     // Throws CryptographicException
-    private X509Certificate2? GetClientCertificate(IntPtr baseAddress, HttpApiTypes.HTTP_REQUEST_V2* nativeRequest)
+    private X509Certificate2? GetClientCertificate(IntPtr baseAddress, HTTP_REQUEST_V2* nativeRequest)
     {
-        var request = nativeRequest->Request;
-        long fixup = (byte*)nativeRequest - (byte*)baseAddress;
+        var request = nativeRequest->Base;
+        var fixup = (byte*)nativeRequest - (byte*)baseAddress;
```

Comment:
I don't see a code style doc in our repo, are we following the one in the runtime repo? where var is allowed only when the RHS is a new or a cast so it's explicit.

Reply:
AspNetCore's style guide is the opposite, use var unless required otherwise. 🤷 Most of this is codified in our repo settings.
https://github.com/dotnet/aspnetcore/blob/ee6d623741948c6ca907f60bad06736229b7d5e3/.editorconfig#L36-L39